src/components/bannerTitle/_spec.js

```diff
@@ -1,24 +1,27 @@
 import React from 'react'
-import renderer from 'react-test-renderer'
 import 'jest-styled-components'
 import { shallow } from 'enzyme'
+import theme from '../../theme/theme'
 
 import BannerTitle from './'
 
 describe('BannerTitle', () => {
   it('should render', () => {
-    const tree = renderer.create(<BannerTitle />)
+    const tree = shallow(<BannerTitle />)
     expect(tree).toMatchSnapshot()
   })
 
-  it('should render a <span />', () => {
-    const wrapper = shallow(<BannerTitle />)
-    expect(wrapper.find('span')).toHaveLength(1)
+  it('should render a <h1 />', () => {
+    const wrapper = shallow(<BannerTitle>foo</BannerTitle>)
+    expect(wrapper.find('TestBannerTitle')).toHaveLength(1)
   })
 
   it('should render child text ', () => {
     const text = 'Here is a test title!'
-    const wrapper = shallow(<BannerTitle>{text}</BannerTitle>)
-    expect(wrapper.find('span').text()).toBe(text)
+    const wrapper = shallow(<BannerTitle>{text}</BannerTitle>, {
+      context: { theme },
+    })
+    // Currently having issues testing with enzyme as the theme does not get provided to the styled componentw
```

Comment:
is this still an issue now that the snapshots have been updated? is it worth deleting this comment?

Reply:
No this is a separate issue, we're only using snapshots here since the other tests seem to be having issues with styled component props